src/postgraphile/withPostGraphileContext.ts

```diff
@@ -240,6 +247,16 @@ async function setupPgClientTransaction({
     await pgClient.query(query)
   }
 
+  if (typeof preRequest === 'string') {
+    const query = sql.compile(sql.query`select ${sql.raw(`${preRequest}()`)}`)
```

Comment:
Just use:

```js
const query = `select ${preRequest}()`
```

(That way we can avoid using `sql.raw`, plus it's faster since it doesn't need to compile from an AST.)